scalding-serialization/src/main/scala/com/twitter/scalding/serialization/macros/impl/OrderedBufferableProviderImpl.scala

```diff
@@ -16,7 +16,7 @@
 package com.twitter.scalding.serialization.macros.impl
 
 import scala.language.experimental.macros
-import scala.reflect.macros.Context
+import scala.reflect.macros.whitebox.Context
```

Comment:
Please undo non scala 2.10 compatible changes, they don't seem like they should be in this PR anyway